cukinia

```diff
@@ -811,6 +812,15 @@ _cukinia_gpio()
 	return $ret
 }
 
+# _cukinia_device: Check that device exists
+# arg1: the device sysfs path
+_cukinia_device()
```

Comment:
I see no value in this function, over cukinia_test -d <same_argument>.. What problem does it solve ?

Reply:
you are right, as this is not checking more than the presence of the device directory, cukinia_test -d <same_argument>. is more relevant.